docs/install-overview.md

```diff
@@ -6,6 +6,16 @@ In version 0.7.0 and later, you can run Ark in any namespace, which requires add
 
 In version 0.9.0 and later, you can use Ark's integration with restic, which requires additional setup. See [restic instructions][20].
 
+## Customize configuration
+
+Whether you run Ark on a cloud provider or on-premises, you can specify the name of the default volume snapshot when you set up the server by running this additional command:
```

Comment:
I would say "...on-premises, if you have more than one volume snapshot location for a given PV provider, you can specify the one to be used by default when taking backups by setting a server flag in your Ark deployment YAML:"